### PR TITLE
Use cassandra instead of mysql for clusterdomain migration.

### PR DESCRIPTION
**Pull request by Rohit-PX**
Signed-off-by: Rohit-PX <rohit@portworx.com>

**What type of PR is this?**
integration-test

**What this PR does / why we need it**:
Switch cluster domain migration test to cassandra from mysql

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
no

**Does this change need to be cherry-picked to a release branch?**:
no

### CODE REVIEW

**Comment by disrani-px** [comment]
The scale factor is being set to 1, it should be whatever was at the source

**Comment by Rohit-PX** [comment]
> The scale factor is being set to 1, it should be whatever was at the source

Creating a copy of the old scale factor map and restoring from that now.

File: test/integration_test/clusterdomain_test.go [hunk 1]
```diff
@@ -146,6 +146,10 @@ func testClusterDomainsFailover(
 	// Reduce the replicas on cluster 1
 	scaleFactor, err := schedulerDriver.GetScaleFactorMap(ctxs[0])
 	require.NoError(t, err, "Unexpected error on GetScaleFactorMap")
+
+	// Copy the old scale factor map
+	oldScaleFactor := scaleFactor
```
Comment (disrani-px) [inline test/integration_test/clusterdomain_test.go]: This won't make a copy of the map, so oldScaleFactor will also have 0s for the scale after the for loop below

Comment (Rohit-PX) [reply test/integration_test/clusterdomain_test.go]: Fixed.

File: test/integration_test/clusterdomain_test.go [hunk 2]
```diff
@@ -156,11 +163,11 @@ func testClusterDomainsFailover(
 	err = setRemoteConfig(remoteFilePath)
 	require.NoError(t, err, "Error setting remote config")
 
-	// Increase the replicas on cluster 1
+	// Set scale factor to it's orignal values on cluster 1
 	scaleFactor, err = schedulerDriver.GetScaleFactorMap(preMigrationCtx)
 	require.NoError(t, err, "Unexpected error on GetScaleFactorMap")
 	for k := range scaleFactor {
```
Comment (disrani-px) [inline test/integration_test/clusterdomain_test.go]: Don't need this loop, just pass in oldScaleFactor to scale up

Comment (Rohit-PX) [reply test/integration_test/clusterdomain_test.go]: Fixed.

**Pull request merged**